transcriptic/commands.py

```diff
@@ -1489,6 +1489,25 @@ def execute(
     partition_horizon,
     partitioning_swap_device_id,
 ):
```

Comment:
Seems good. You could also support (or not?) https. I don't think we have the requirement to support it, but you could also warn if the user accidentally uses https protocol.